proto/controller.proto

```diff
@@ -24,14 +24,87 @@ service TopologyManager {
   rpc CreateTopology (CreateTopologyRequest) returns(CreateTopologyResponse) {}
   rpc DeleteTopology (DeleteTopologyRequest) returns(DeleteTopologyResponse) {}
   rpc ShowTopology (ShowTopologyRequest) returns(ShowTopologyResponse) {}
+  rpc CreateCluster (CreateClusterRequest) returns(CreateClusterResponse) {}
```

Comment:
All of these RPCs should have a doc string one liner

Reply:
Done